### PR TITLE
Feature/android/1315/directions detailed guide

### PR DESCRIPTION
**Pull request by timMikkelsen**

### CODE REVIEW

File: src/android/v3/guides/directions/directions-wayfinding-instructions.md [hunk 1]
```diff
@@ -0,0 +1,664 @@
...
+  title: Get Directions and Show Wayfinding Instructions
+  key: android-v3-guides-get-directions-and-show-wayfinding
+  order: 410
+---
+
+In this tutorial we will show how to work with the route model returned from a directions service call. We will also show how you can utilize interaction between the route rendering on the map and textual instructions showed in another view.
+
+To start we will use the complete Getting started example found here: [Java](https://github.com/MapsPeople/MapsIndoors-Getting-Started-Android) or [Kotlin](https://github.com/MapsPeople/MapsIndoors-Getting-Started-Android-Kotlin)
+
+We will work with changing the implementation of the `NavigationFragment` and the `RouteLegFragment`.
```
Comment (JG1995) [inline src/android/v3/guides/directions/directions-wayfinding-instructions.md:14]:
```suggestion
You will work with changing the implementation of the `NavigationFragment` and the `RouteLegFragment`.
```

File: src/android/v3/guides/directions/directions-wayfinding-instructions.md [hunk 2]
```diff
@@ -0,0 +1,664 @@
...
+        fragment.mMapsActivity = mapsActivity
+        return fragment
+    }
+}
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+Now we will replace the old view in the `RouteLegFragment`.
```
Comment (JG1995) [inline src/android/v3/guides/directions/directions-wayfinding-instructions.md:61]:
```suggestion
Next step is replacing the old view in the `RouteLegFragment`.
```

File: src/android/v3/guides/directions/directions-wayfinding-instructions.md [hunk 3]
```diff
@@ -0,0 +1,664 @@
...
+        mMapsActivity!!.removeFragmentFromBottomSheet(this)
+        mMapsActivity!!.getMpDirectionsRenderer()?.clear()
+    }
+}
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+We will now change the `ViewPager` implementation to contain the revised `RouteLegFragment` for each leg of the route.
```
Comment (JG1995) [inline src/android/v3/guides/directions/directions-wayfinding-instructions.md:143]:
```suggestion
Next step is to change the `ViewPager` implementation to contain the revised `RouteLegFragment` for each leg of the route.
```

File: src/android/v3/guides/directions/directions-wayfinding-instructions.md [hunk 4]
```diff
@@ -0,0 +1,664 @@
...
+    fragment.mDistance = distance;
+    fragment.mDuration = duration;
+    return fragment;
+}
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+We will now update the `onViewCreated` method to use the new views added earlier in the tutorial.
```
Comment (JG1995) [inline src/android/v3/guides/directions/directions-wayfinding-instructions.md:387]:
```suggestion
You must also update the `onViewCreated` method to use the new views added earlier in the tutorial.
```

**Review by JG1995 — CHANGES_REQUESTED** [review]
This is mostly wording changes and grammar fixes, but give them a read to make sure I didn't accidentally change the meaning of the descriptions in my rewording.

File: src/android/v3/guides/directions/directions-renderer.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+    mpDirectionsRenderer?.previousLeg()
+}
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+## Show Content of Nearby Locations
+
+It is possible to show contextual information on the end points of the rendered path of a route segment by configuring the directions renderer to look for nearby Locations or POIs.
```
[the remark this answers is not in the document]
Comment (timMikkelsen) [reply src/android/v3/guides/directions/directions-renderer.md:190]: This is not correct. It is only the marker/label on the map that is changed. Directions remain unchanged.

File: src/android/v3/guides/directions/directions-renderer.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+mDirectionsRenderer?.useContentOfNearbyLocations(MPContextualInfoSettings.Builder()
+            .setMaxDistance(30.0)
+            .setTypes(Collections.singletonList("entries"))
+            .build())
+```
+
+</mi-tab-panel>
+</mi-tabs>
+
+The defaults of the `ContextualInfoSettings` builder is `maxDistance` at 5 meters and the `ContextualInfoScope` as icon and name. No Types or Categories is set as default. Not applying any will make it search through all Locations to use as contextual information.
```
[the remark this answers is not in the document]
Comment (timMikkelsen) [reply src/android/v3/guides/directions/directions-renderer.md:223]: Types _or_ Categories? 😄

File: src/android/v3/guides/directions/directions-renderer.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (JG1995) [reply src/android/v3/guides/directions/directions-renderer.md:190]: Alright, ignore this suggestion then.

**Review by JG1995 — APPROVED** [review]

**Pull request merged**